.github/workflows/main.yml

```diff
@@ -140,6 +140,50 @@ jobs:
           ./demo/run-osm-demo.sh
           go run ./ci/cmd/maestro.go
 
+  integration-tresor-with-same-sa:
```

Comment:
Instead of adding another scenario which is going to increase the CI time, we should simply change one of the existing 3 scenarios to deploy services in the same service account.

Reply:
I addressed my own comment.